### ISSUE TITLE
Multi Calendars - One Instance - Overlaid ?

### ISSUE THREAD

**Issue by TLFulton** [mscrivo/OotD#104]
How can we make it so that I can select more than one calendar in the "Select Folder" option for which calendar to display, and overlay them in one instance on my desktop calendar? This has to be a programming feature, as I can access all of my calendars, but can only display one of them at a time. I want to be able to overlay them like in Outlook, can that feature be mimicked by OotD?

The only other solution I've encountered is a third party application that merges all calendars into one folder, but that has fallbacks of its own; like it doesn't auto sync and the calendar colors are lost in the process.

**Comment by TLFulton** [comment]
Yes, Code2 doesn't work well because it backfeeds the sync so each entry doubles every time you sync. It's a real hassle. As is, OotD just doesn't work well for people who have multiple calendars they want to link up.

**Comment by udobehm** [comment]
Hi, Cal4Desktop offers the function to show several calendars overlaid on the desktop (see https://www.somebytes.de/produkte/cal4desktop/). Why is not possible to program that with your program?? I absolutely need this function :((
